lib/src/inline_parser.dart

```diff
@@ -938,26 +958,50 @@ class TagSyntax extends InlineSyntax {
   /// [getChildren], in which [parser] parses any nested text into child nodes.
   /// The returned [Node] incorpororates these child nodes.
   Node? close(InlineParser parser, Delimiter opener, Delimiter closer,
-      {required List<Node> Function() getChildren}) {
-    var strong = opener.length >= 2 && closer.length >= 2;
-    return Element(strong ? 'strong' : 'em', getChildren());
+      {required String tag, required List<Node> Function() getChildren}) {
```

Comment:
Style nit: please add a trailing comma here, so that each parameter appears on its own line.

Reply:
Fixed. I thought we prefer to use this no trailing comma style.

Reply:
I actually don't have a strong preference. Except when we have some long parameters (like with the required keyword), and now they're creeping onto a third line. In the analyzer codebase we've been moving towards using trailing commas more. It's not a hard rule yet.